lib/cassandra/column_container.rb

```diff
@@ -127,16 +127,15 @@ def to_cql
 
         options << 'COMPACT STORAGE' if @compact_storage
         unless @bloom_filter_fp_chance.nil?
-          options <<
-            "bloom_filter_fp_chance = #{Util.encode_object(@bloom_filter_fp_chance)}"
+          options << "bloom_filter_fp_chance = #{Util.encode_object(@bloom_filter_fp_chance)}"
         end
         options << "caching = #{Util.encode_object(@caching)}" unless @caching.nil?
         options << "comment = #{Util.encode_object(@comment)}" unless @comment.nil?
         options << "compaction = #{@compaction_strategy.to_cql}" unless @compaction_strategy.nil?
         options << "compression = #{Util.encode_object(@compression)}" unless @compression.nil?
+        options << "crc_check_chance = #{Util.encode_object(@crc_check_chance)}" unless @crc_check_chance.nil?
         unless @local_read_repair_chance.nil?
```

Comment:
I'm trying to stick to a 120 character line length. Makes it easier to look at side-by-side diffs on my laptop screen.